src/main/java/com/aws/greengrass/util/FileSystemPermission.java

```diff
@@ -70,6 +82,90 @@
         return ret;
     }
 
+    /**
+     * Convert to a list of Acl entries for use with AclFileAttributeView.setAcl.
+     *
+     * @param path path to apply to
+     * @return List of Acl entries
+     * @throws IOException if any exception occurs while converting to Acl
+     */
+    public List<AclEntry> toAclEntries(Path path) throws IOException {
```

Comment:
move this to the windows implementation since this is entirely platform specific